PowerShell/JumpCloud Module/Public/Commands/Get-JCCommand.ps1

```diff
@@ -155,28 +155,26 @@ CommandID has an Alias of _id. This means you can leverage the PowerShell pipeli
 
                         $Value = ($param.value).replace('*', '')
 
-                        if (($param.Value -match '.+?\*$') -and ($param.Value -match '^\*.+?'))
-                        {
+                        if (($param.Value -match '.+?\*$') -and ($param.Value -match '^\*.+?')) {
                             # Front and back wildcard
-                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "$Value" })
-                        }
-                        elseif ($param.Value -match '.+?\*$')
-                        {
+                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "(?i)$Value" })
+                        } elseif ($param.Value -match '.+?\*$') {
                             # Back wildcard
-                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "^$Value" })
-                        }
-                        elseif ($param.Value -match '^\*.+?')
-                        {
+                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "(?i)^$Value" })
+                        } elseif ($param.Value -match '^\*.+?') {
                             # Front wild card
-                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "$Value`$" })
```

Comment:
![Screen Shot 2022-07-14 at 3 15 48 PM](https://user-images.githubusercontent.com/54448601/179087597-0ca8aa6a-eef4-4b43-81af-995e52d0efd3.png)

I'm thinking we missed something here:

We are not regex filtering on the wild card cases

I think we can just do:
`if (($param.Value -match '.+?\*$') -and ($param.Value -match '^\*.+?')) {
                            # Front and back wildcard
                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "(?i)$([regex]::Escape($Value))" })
                        } elseif ($param.Value -match '.+?\*$') {
                            # Back wildcard
                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "(?i)^$([regex]::Escape($Value))" })
                        } elseif ($param.Value -match '^\*.+?') {
                            # Front wild card
                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "(?i)$([regex]::Escape($Value))`$" })
                        } elseif ($param.Value -match '^[-+]?\d+$') {
                            # Check for integer value
                            (($Search.filter).GetEnumerator()).add($param.Key, $([regex]::Escape($Value)))
                        } else {
                            (($Search.filter).GetEnumerator()).add($param.Key, @{'$regex' = "(?i)(^$([regex]::Escape($Value))`$)" })
                        }`